lib/README.md

```diff
@@ -3,9 +3,6 @@ referred to as the Common Open Promise Body Library.
 
 Layout of this `lib` directory:
 
-* `3.6/*.cf`: Modularized version of the CFEngine Standard Library for
```

Comment:
how is this related to 3.12? Did we make a switch from 3.6/*.cf to lib/*.cf at that point?

Reply:
or was 3.6 supported when 3.12 was released?

Reply:
It's not related to 3.12 at all, it's just another cleanup that I found while I was looking around. This is back from before we had macros to guard against future syntax and because of that we split the stdlib into version specific trees so that there were no new functions loaded by old agents that didn't understand the macros to avoid parsing functions and language features they didn't know about. 

It seemed too small to have in a separate PR, that's why its in this PR in a separate commit and that commit doesn't have a Ticket keyword :)